package.json

```diff
@@ -60,8 +61,8 @@
   "license": "MIT",
   "lint-staged": {
     "*.{cjs,js,ts,css,scss,svelte}": [
-      "eslint --fix",
-      "prettier --write --plugin-search-dir=."
+      "npx eslint --ignore-path .gitignore --fix .",
+      "npm run format"
```

Comment:
I think we shouldn't run the scripts here. Prettier should only run for the changed files, while the script should run for all files.

Reply:
Done